### PR TITLE
Adding Section 1: Introduction, Section 4: BPI Abstraction Layers and Section 7: BPI Storage

### PR DESCRIPTION
**Pull request by Therecanbeonlyone1969**
**Section 1: Introduction**
* Overview
* Glossary (no updates from its previous version)
* Typographical Conventions (no updates from its previous version -> Can we remove it?)

**Section 4: BPI Abstraction Layers** with definitions, concepts, scope, and security considerations for BPI Abstraction Layers
* BPI Abstraction Scope and Components
* BPI Abstraction Layer Security and Integration

**Section 7: General BPI Storage Capabilities** with definitions, key concepts, and overviews of BPI Storage components applicable to all BPI layers and necessary in a compliant Baseline Protocol Implementation as delineated in the following subsections 
* BPI Storage Security
* BPI Storage Privacy
* BPI Data Orchestration
* BPI-External Storage: Edge Storage
* BPI-Internal Storage

### CODE REVIEW

File: core/baseline-core-v1.0-psd01.md [hunk 1]
```diff
@@ -566,195 +622,877 @@ Below we list and define the components of each layer. The detailed requirements
...
-| Requirement ID|Requirement  |
-| :--- | :--- |
-| PRICON1 | something MUST something  |
-| PRICON2 | something SHOULD something |
+**[CR13]>[D12]** A CDN utilized in a BPI Abstraction Layer and operated by a 3rd party MUST support BPI user specific and time based content access control. 
 
+**[CR14]>[D12]** A CDN utilized in a BPI Abstraction Layer and operated by a 3rd party MUST support time based, automated content removal.
 
-## 5.2 Confidentiality
+**[R61]**	BPI Abstraction Layers MUST support integration with internal, as defined in [Section 5](#5-Middleware-Communication-and-Interoperability), and/or external BPI User identity access management (IAM) or identity provider (IdP) systems.
```
Comment (mehranshakeri) [inline core/baseline-core-v1.0-psd01.md:870]: link broken?

File: core/baseline-core-v1.0-psd01.md [hunk 2]
```diff
@@ -566,195 +622,877 @@ Below we list and define the components of each layer. The detailed requirements
...
+
+Smart Contract management comprises full lifecycle management from testing, initial deployments, update and deactivation (for CCSM Abstraction Layer only). 
+
+The Figure 6 below shows the reference architecture for a BPI or CCSM Abstraction Layer. 
+
+<figure>
+  <img
+  src="./images/Baseline-Fig2-BPI-Abstraction-Layer.png"
+  >
+  <figcaption>Figure 6: High-Level Reference Architecture of a BPI Layer.</figcaption>
```
Comment (mehranshakeri) [inline core/baseline-core-v1.0-psd01.md:888]: Please provide legends to explain colors

File: core/baseline-core-v1.0-psd01.md [hunk 3]
```diff
@@ -566,195 +622,877 @@ Below we list and define the components of each layer. The detailed requirements
...
+</figure>
+
+
+## 4.2 BPI Abstraction Layer Security and Integration
+
+The security requirements of this section are distinct from the security requirements of the other BPI layers or any custom APIs because the external systems which are invoking services exposed by the BPI or CCSM Abstraction Layer should not be assumed to be a trusted service without authentication. This is because this standard does not define the operating model of external systems or a BPI or any of the BPI layers, and, therefore, must necessarily prescribe requirements assuming a 100% adversarial environment.
+
+**[R65]**	Abstraction Layers utilized in a BPI MUST be compatible with widely used external authentication services or .
```
Comment (mehranshakeri) [inline core/baseline-core-v1.0-psd01.md:896]: Incomplete sentence! `or .`

File: core/baseline-core-v1.0-psd01.md (hunk 2 above)
Comment (mehranshakeri) [reply core/baseline-core-v1.0-psd01.md:888]: Or all black and white if they are not grouped by color.

File: core/baseline-core-v1.0-psd01.md [hunk 4]
```diff
@@ -566,195 +622,877 @@ Below we list and define the components of each layer. The detailed requirements
...
+
+Weak synchrony in this context means, 
+* that all messages will eventually reach their intended recipients and 
+* that after a certain, yet unknown, time the network will become synchronous again.
+
+**[RXX]**	Replication conflicts MUST be automatically detectable 
+
+**[RXX]** Replication conflict MUST be resolvable either automatically or manually.
+
+**[RXX]**	BPI Edge Storage MUST use a secure and privacy-preserving wire protocol.
```
Comment (mehranshakeri) [inline core/baseline-core-v1.0-psd01.md:1421]: Copy past of other ones. Shall we remove or improve it?

File: core/baseline-core-v1.0-psd01.md [hunk 5]
```diff
@@ -566,195 +622,877 @@ Below we list and define the components of each layer. The detailed requirements
...
+
+**[RXX]** Data in transit in a BPI MUST be encrypted.
+
+**[RXX]** Data at rest in a BPI MUST be encrypted.
+
+**[RXX]**	BPI Storage arranged in a network MUST support pairwise key/identity-relationships between storage nodes.
+
+This is also known as secure connection.
+
+**[RXX]**	BPI Storage MUST be compatible with widely used external authentication services or .
```
Comment (mehranshakeri) [inline core/baseline-core-v1.0-psd01.md:1346]: Copy past of other ones. Shall we remove or improve it?

File: core/baseline-core-v1.0-psd01.md [hunk 6]
```diff
@@ -566,195 +622,878 @@ Below we list and define the components of each layer. The detailed requirements
...
+
+**[CRXX]>[O3]** Partially Persistent BPI Storage  MUST be append-only.
+
+Non-normative examples of such data bases are but not limited to OracleDB, MongoDB, PostGres, Cassandra and DynamoDB.
+
+### 7.5.2 BPI Storage: Decentralized Deployment
+
+**[RXX]**	BPI Storage  MUST support authenticated naming systems.
+
+An authenticated naming system in the context of this document is defined ...
```
Comment (mehranshakeri) [inline core/baseline-core-v1.0-psd01.md:1458]: incomplete

**Review by Consianimis — APPROVED** [review]

File: core/baseline-core-v1.0-psd01.md (hunk 2 above)
Comment (Therecanbeonlyone1969) [reply core/baseline-core-v1.0-psd01.md:889]: b&w figure now

File: core/baseline-core-v1.0-psd01.md (hunk 3 above)
Comment (Therecanbeonlyone1969) [reply core/baseline-core-v1.0-psd01.md]: done

File: core/baseline-core-v1.0-psd01.md (hunk 6 above)
Comment (Therecanbeonlyone1969) [reply core/baseline-core-v1.0-psd01.md]: updated

File: core/baseline-core-v1.0-psd01.md (hunk 1 above)
Comment (Therecanbeonlyone1969) [reply core/baseline-core-v1.0-psd01.md:871]: seems to be working ... at least now

File: core/baseline-core-v1.0-psd01.md (hunk 5 above)
Comment (Therecanbeonlyone1969) [reply core/baseline-core-v1.0-psd01.md:1347]: that is intentional ... because it is specific to BPI Storage

**Comment by Therecanbeonlyone1969** [comment]
Closing pull request since this PR is subsumed in PR #59.

**Pull request closed without merging**